backend/Utils/EmailUtils.ts

```diff
@@ -2,9 +2,32 @@ import google from "googleapis";
 import path from "path";
 import { encode } from "js-base64";
 import dotenv from "dotenv";
+import jwt from "jsonwebtoken";
 
-async function makeEmail({ to, from, subject }) {
-  const message = `This is a test email for reset password.`;
+dotenv.config();
+const { sign, verify } = jwt;
+
+const baseUrl = process.env.PASSWORD_RESET_BASE_URL || "https://localhost:7600";
+
+async function makeEmail({ to, from, subject }, token) {
```

Comment:
I think I was mistaken when we talked about this this morning - we do need to keep this function here in EmailUtils. But it should not be hard coded to the reset password email. Instead, the `body` should be passed in as a string to this function, and the reset password email body itself can go into ResetPassword.ts rather than here.